README.md

```diff
@@ -18,6 +18,8 @@ given below:
 
 ### Advance
```

Comment:
@Savidude this should be Advanced, IMO.

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/187/commits/bfec31c1d39a0f2fcfe31a0280bbeaf4922b7112